packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -1043,3 +1044,35 @@ class ExcludeFocus extends StatelessWidget {
     );
   }
 }
+
+/// A focus area is used to expand a focus node as part of the render object layer
+/// outside of its descendant render objects.
```

Comment:
We should also explain what is the use case, i.e. why would developers want to use this widget